server/game/cards/01-DTR/SanfordTaylor.js

```diff
@@ -0,0 +1,34 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class SanfordTaylor extends DudeCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Noon: Sanford Taylor',
+            playType: ['noon'],
+            condition: () => this.locationCard.getType() === 'deed', 

```

Comment:
This work too, but let's use a function since it is there :)

```suggestion
            condition: () => this.isAtDeed(), 
```